cartographer_ros/cartographer_ros/cartographer_grpc/node_grpc_main.cc

```diff
@@ -43,9 +42,13 @@ DEFINE_string(
     "If non-empty, serialize state and write it to disk before shutting down.");
 DEFINE_string(load_state_filename, "",
               "If non-empty, filename of a .pbstream file "
-              "to load, containing a saved SLAM state.");
+              "to load, containing a saved SLAM state. "
+              "Unless --forward_local_state_file is set, the filepath refers "
+              "to the gRPC server's file system.");
 DEFINE_bool(load_frozen_state, true,
             "Load the saved state as frozen (non-optimized) trajectories.");
+DEFINE_bool(forward_local_state_file, false,
+            "Stream the saved state from a local path to the (remote) server.");
```

Comment:
I'd find "Upload the .pbstream file from local path to the (remote) gRPC server instead a loading the saved state from the server file system." easier to read.